clouddriver-core/src/test/groovy/com/netflix/spinnaker/clouddriver/orchestration/DefaultOrchestrationProcessorSpec.groovy

```diff
@@ -108,11 +108,12 @@ class DefaultOrchestrationProcessorSpec extends Specification {
     where:
     exception                     | sagaId               || retryable
     new RuntimeException()        | null                 || false
-    new NonRetryableException()   | null                 || false
-    new RuntimeException()        | new SagaId("a", "a") || true
+    new RetryableException()      | null                 || false
```

Comment:
nit: this is a bit confusing now... maybe add a comment that the retry logic is only allowed for saga based operations?

Reply:
Added a comment.